praw/models/reddit/subreddit.py

```diff
@@ -611,6 +612,8 @@ def _submit_media(self, data, timeout, without_websockets):
                 "the original exception.",
                 ws_exception,
             )
+        if ws_update.get("type", "") == "failed":
```

Comment:
Is the second argument necessary to `get`? Typically `None` is returned in such instances, so we don't need to provide a second argument unless we want a specific default.